src/app/(profile)/(components)/ProfileList/ProfileList.tsx

```diff
@@ -20,7 +21,11 @@ const ProfileList = () => {
     <div className={styles.profileCards}>
       <div className={styles.profileListCont}>
         {profiles.map((profile) => (
-          <ProfileListItem key={profile.id} data={profile} />
+          <ProfileListItem
+            key={profile.id}
+            data={profile}
+            href={`${AppRoutes.profile}/${profile.githubUsername}`}
```

Comment:
I think you do not need to pass it here. You can get `githubUsername` inside this component from data :)